specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2020-07-01/appplatform.json

```diff
@@ -87,14 +87,14 @@
           }
         ],
         "responses": {
-          "201": {
-            "description": "Created. The response describes the new Service and contains a Location header to query the\r\noperation result.",
+          "200": {
```

Comment:
This seems to have nothing to do with setting `modelAsString=false` and is a breaking change.

Reply:
it is just reordering the keys, it is still the same responses. This is due to JSON.parse somehow only deciding to sort and reorder number like keys.